### DISCUSSION TITLE
Lists under Frequency Manager have dissapeared

### DISCUSSION THREAD

**Discussion by idrankwhat** [AlexandreRouma/SDRPlusPlus#874] [Q&A]
How do I get the lists I've made back in Frequency Manager.
Had lists for:
HF
VHF AIR
UHF AIR
Railroad
etc. but they have all vanished.

**Comment by AlexandreRouma** [comment]
Not sure, can be a lot of things but you haven't given much info about what happened before the issue shows up, what system it's running on, etc.
If a crash occurs as the software is trying to save it's configuration it's usually lost.
I recommend next time you back up the file that contains those bookmarks (location depends on your OS)
